static/dtale/menu/DataViewerMenu.jsx

```diff
@@ -53,207 +54,212 @@ class ReactDataViewerMenu extends React.Component {
+        };
     return (
-      <div
-        className="column-toggle__dropdown"
-        hidden={!menuOpen}
-        style={{ minWidth: "13.65em", top: "1em", left: "0.5em" }}>
-        {menuOpen && <GlobalHotKeys keyMap={{ CLOSE_MENU: "esc" }} handlers={{ CLOSE_MENU: closeMenu }} />}
-        <header className="title-font">D-TALE</header>
-        <ul>
-          <XArrayOption columns={_.reject(this.props.columns, { name: "dtale_index" })} />
-          <DescribeOption open={buttonHandlers.DESCRIBE} />
-          <li className="hoverable">
-            <span className="toggler-action">
-              <button className="btn btn-plain" onClick={buttonHandlers.FILTER}>
-                <i className="fa fa-filter ml-2 mr-4" />
-                <span className="font-weight-bold">Custom Filter</span>
-              </button>
-            </span>
-            <div className="hoverable__content menu-description">{Descriptions.filter}</div>
-          </li>
-          <li className="hoverable">
-            <span className="toggler-action">
-              <button className="btn btn-plain" onClick={buttonHandlers.BUILD}>
-                <i className="ico-build" />
-                <span className="font-weight-bold">Build Column</span>
-              </button>
-            </span>
-            <div className="hoverable__content menu-description">{Descriptions.build}</div>
-          </li>
-          <MergeOption open={() => window.open(menuFuncs.fullPath("/dtale/popup/merge"), "_blank")} />
-          <li className="hoverable">
-            <span className="toggler-action">
-              <button className="btn btn-plain" onClick={openPopup("reshape", 400, 770)}>
-                <i className="fas fa-tools ml-2 mr-4" />
-                <span className="font-weight-bold">Summarize Data</span>
-              </button>
-            </span>
-            <div className="hoverable__content menu-description">{Descriptions.reshape}</div>
-          </li>
-          <DuplicatesOption open={buttonHandlers.DUPLICATES} />
-          <li className="hoverable">
-            <span className="toggler-action">
-              <button className="btn btn-plain" onClick={openTab("correlations")}>
-                <i className="ico-bubble-chart" />
-                <span className="font-weight-bold">Correlations</span>
-              </button>
-            </span>
-            <div className="hoverable__content menu-description">{Descriptions.corr}</div>
-          </li>
-          {(!pythonVersion || (pythonVersion[0] >= 3 && pythonVersion[1] >= 6)) && (
-            <li className="hoverable">
+      <div {...containerProps}>
+        {!menuPinned && menuOpen && (
+          <GlobalHotKeys keyMap={{ CLOSE_MENU: "esc" }} handlers={{ CLOSE_MENU: closeMenu }} />
+        )}
+        <header className="title-font pb-1">D-TALE</header>
+        <div
+          style={{
+            height: `calc(100vh - ${menuPinned ? 40 : 68}px)`,
+            overflowY: "scroll",
```

Comment:
This will solve the horizontal scroll in the menu popup
```suggestion
            overflowY: "scroll",
            overflowX: "hidden",
```